PyInstaller/depend/utils.py

```diff
@@ -80,9 +80,15 @@ def create_py3_base_library(libzip_filename, graph):
                             fc.write(BYTECODE_MAGIC)
                             if is_py37:
                                 # Additional bitfield according to PEP 552
-                                # zero means timestamp based
-                                fc.write(struct.pack('<I', 0))
-                            fc.write(struct.pack('<II', timestamp, size))
+                                # one means hash based but don't check the hash
+                                fc.write(struct.pack('<I', 0b1))
```

Comment:
Since you are using binary notation, what about specifying the "check_source" flag, too (`0b01`) and adjust the comment to something like "0b01 means hash based but don#t check the hash"?

Reply:
Done